packages/manager/apps/web/client/app/private-database/dashboard/private-database.html

```diff
@@ -193,20 +193,66 @@
             ></p>
         </div>
 
-        <div data-ng-controller="PrivateDatabaseTabsCtrl">
-            <wuc-ovh-tabs
-                tabs="tabs"
-                selected-tab="selectedTab"
-                product="privateDatabase"
-                change-tab="setSelectedTab"
-                tr="tr"
-            ></wuc-ovh-tabs>
-            <div class="tab-content">
-                <div
-                    class="tab-pane active"
-                    data-ng-include="!server.isExpired && 'private-database/' + toKebabCase(selectedTab) + '/private-database-' + selectedTab.toLowerCase() + '.html' || ''"
-                ></div>
-            </div>
+        <!-- <div > -->
+        <oui-header-tabs>
+            <oui-header-tabs-item
+                href="{{ ctrl.stateLink }}"
+                active="ctrl.currentActiveLink() === ctrl.stateLink"
+            >
+                <span data-translate="privateDatabase_tab_STATE"></span>
+            </oui-header-tabs-item>
+            <oui-header-tabs-item
+                data-ng-if="database.serviceInfos.status !== 'expired'"
+                href="{{ ctrl.userLink }}"
```

Comment:
maybe one-time binding? and also in other tabs as well.

Reply:
See my previous comment :)